pyproject.toml

```diff
@@ -1,32 +1,274 @@
-[tool.poetry]
+[build-system]
+requires = ["setuptools>=42"]
+build-backend = "setuptools.build_meta"
+
+[project]
 name = "mage-ai"
-version = "0.8.75-dev"
-description = "DevEx poetry configuration"
-authors = ["Mage"]
+version = "0.9.21"
+description = "Mage is a tool for building and deploying data pipelines."
 readme = "README.md"
+requires-python = ">=3.8"
+license = {file = "LICENSE"}
+authors = [
+  {email = "eng@mage.ai"},
+  {name = "Mage"}
+]
+maintainers = [
+  {name = "Mage", email = "eng@mage.ai"}
+]
+classifiers = [
+  "Programming Language :: Python :: 3",
+  "License :: OSI Approved :: Apache Software License",
+  "Operating System :: OS Independent"
+]
 
-[tool.poetry.dependencies]
-python = ">=3.8,<4.0"
-
-[tool.poetry.group.dev.dependencies]
-mypy = "*"
-black = "*"
-flake8 = "*"
-pre-commit = "*"
-pytest = "*"
-pytest-cov = "*"
-interrogate = "*"
-isort = "*"
-commitizen = "*"
-ipykernel = "^6.17.1"
-pandas-stubs = "*"
-flake8-bugbear = "*"
-sourcery = "*"
+dependencies = [
+  'aiofiles==22.1.0',
+  'aiohttp',
+  'alembic>=1.7.5',
+  'async_timeout',
+  'bcrypt==4.0.1',
+  'click',
+  'confluent-avro',
+  'croniter==1.3.7',
+  'cryptography==36.0.2',
+  'dask>=2022.2.0',
+  'Faker==4.14.0',
+  'flask-cors',
+  'flask',
```

Comment:
i don't think we need flask dependencies

Reply:
hmm, i don't think this file is used. let me see if i can remove it so we can remove some dependencies